src/states/elder/mod.rs

```diff
@@ -1438,7 +1438,12 @@ impl Elder {
             // TODO: even if having chain reply based on connected_state,
             // we remove self in targets info and can do same by not
             // chaining us to conn_peer list here?
-            let conn_peers = self.connected_peers();
```

Comment:
hmm...
so previously we use the function from peer_manager, and now it's fetched from the peer_map's forward.keys.
does it mean the source is different now? if so, maybe better change a name to avoid mis-use?

Reply:
This is still the connected peers, and in most case both are the same.
However, the one in peer_map is the one used to get the connection info to actually send the message, so this ensure we use the same valid set.